info.yml

```diff
@@ -6,6 +6,21 @@
 
 # expected fields in the configuration file for this engine
 configuration:
+    # NOTE: The 'sg_software_entity' setting is only temporary and
+    #       will be removed once the Software entity becomes native
+    #       in Shotgun
+    sg_software_entity:
+        type: str
+        description: Shotgun CustomNonProjectEntity designated as the
+                     Software entity until it becomes native.
+        default_value: CustomNonProjectEntity23
+
+    software_entity_config_link:
```

Comment:
This would probably be better handled by a constant in the code. It probably wouldn't be something that a TD would want to configure.